release-notes.md

```diff
@@ -17,7 +27,7 @@ The following updates were made in this release:
 * Fixed a cursor leak.
 * Fixed an issue where the advertising identifier was duplicated in the response to the `MobileCore.getSDKIdentifiers` API.
 * Added support for overriding internal network stack with customer-provided code.
-* Added a new interface with failure callback, `AdobeCallbackWithError`, which can be used with the `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities` methods. 
+* Added a new interface with failure callback, `AdobeCallbackWithError`, which can be used with the `MobileCore.getPrivacyStatus`,  `MobileCore.getSdkIdentities` methods.
```

Comment:
Please replace the comma between `MobileCore.getPrivacyStatus` and `MobileCore.getSdkIdentities` with "and", so like this:
"....with the `MobileCore.getPrivacyStatus` and `MobileCore.getSdkIdentities` methods."